testcases/OpTestLPM.py

```diff
@@ -116,10 +117,10 @@ def rmc_service_start(self, mg_system, remote_hmc=None):
         '''
         for svc in ["-z", "-A", "-p"]:
             self.cv_HOST.host_run_command('/opt/rsct/bin/rmcctrl %s' % svc, timeout=120)
-        if not OpTestUtil().wait_for(self.is_RMCActive, timeout=60, args=[mg_system, remote_hmc]):
+        if not self.util.wait_for(self.is_RMCActive, timeout=60, args=[mg_system, remote_hmc]):
```

Comment:
this commit should be merged with this 654aa4af9a90e9da9ce2c6d11a0a02710438d83f or d6e2d31115e0380f3bb6b82e2a6489cf807ce645